src/NzbDrone.Api/Config/HostConfigResource.cs

```diff
@@ -27,11 +27,13 @@ public class HostConfigResource : RestResource
         public bool UpdateAutomatically { get; set; }
         public UpdateMechanism UpdateMechanism { get; set; }
         public string UpdateScriptPath { get; set; }
-        public UseProxy ProxyEnabled { get; set; }
+        public bool ProxyEnabled { get; set; }
         public ProxyType ProxyType { get; set; }
         public string ProxyHostname { get; set; }
         public int ProxyPort { get; set; }
         public string ProxyUsername { get; set; }
         public string ProxyPassword { get; set; }
+        public string ProxySubnetFilter { get; set; }
```

Comment:
WebProxy.BypassList isn't really 'subnet' filtering. it basically filters based on regex/glob (unsure which). So it's probably a good idea to rename it to something like ProxyBypassFilter.

_The current implementation should be sufficient, but if I wanted to make it perfect I would do it a bit differently and support hostnames, address and subnet filtering. eg: "*.sonarr.tv;localhost;127.0.0.1;10.0.0.0/24;192.168.0.0/24"_